express-api/src/services/geocoder/geocoderService.ts

```diff
@@ -34,6 +34,8 @@ export const getSiteAddresses = async (
   url.searchParams.append('addressString', address);
   url.searchParams.append('minScore', minScore);
   url.searchParams.append('maxResults', maxResults);
+  url.searchParams.append('locationDescriptor', 'accessPoint');
+  url.searchParams.append('setBack', '10');
```

Comment:
Was curious about what this locationDescriptor did, so I did some tests with it on/off:
All tests done with 467 Lampson St.
With lines 37 and 38 commented out, I get this:
![image](https://github.com/user-attachments/assets/44309655-029a-4eb7-a837-f5e715d96dee)

With line 38 commented out only, I get this:
![image](https://github.com/user-attachments/assets/45ab1a92-d416-434b-bfd3-a8e8be06b55b)

And with both lines present, I get this:
![image](https://github.com/user-attachments/assets/1cf3040c-2681-4f3e-bab6-b07caf766b70)

It seems like adding these lines might have reduced the accuracy of the location somehow.

Yet, with 1407 Haultain St (a weird split parcel), I get this with them both active:
![image](https://github.com/user-attachments/assets/9963cbfa-ecdf-489e-bd40-60705ebffd14)

And this with neither active:
![image](https://github.com/user-attachments/assets/6d88dc0c-68c1-46ea-bfff-c94dffd21553)

With them, accuracy was 2/2 checked. Without, it was 1/2. so I think it's still worth it.

Reply:
It really could go either way - it just depends on each parcel. But I found the same, on parcels that geocoder sends a good point it makes the point worse but still acceptable, on parcels that geocoder sends a bad point it makes it better.